example-mlem-get-started/code/README.md

```diff
@@ -27,8 +27,8 @@ Now let's install the requirements. But before we do that, we **strongly**
 recommend creating a virtual environment:
 
 ```console
-$ python3 -m venv .env
-$ source .env/bin/activate
+$ python3 -m venv .venv
```

Comment:
minor: better update in a separate PR

Reply:
Why? Didn't get the point.

Reply:
Sorry. it seems to be related to a different example repo? Anyways, it's a minor one. Nw.

Reply:
Ah! Got it. 

> > minor: .venv / .env - in different places it's not consistent, probably we'd like to have .venv everywhere.

I just went ahead and fixed it here, for brevity.